Framework/Core/src/CommonServices.cxx

```diff
@@ -426,6 +428,17 @@ o2::framework::ServiceSpec CommonServices::controlSpec()
     .kind = ServiceKind::Serial};
 }
 
+o2::framework::ServiceSpec CommonServices::spySpec()
+{
+  return ServiceSpec{
+    .name = "spy",
+    .init = [](ServiceRegistry& services, DeviceState& state, fair::mq::ProgOptions& options) -> ServiceHandle {
+      return ServiceHandle{TypeIdHelpers::uniqueId<SpyService>(), new SpyService(services, state)};
+    },
+    .configure = noConfiguration(),
```

Comment:
there is now the ability to specify wether a device is enabled by default. You should probably mark yours as:

```c++
.active = false
```